src/generate_race_var.ado

```diff
@@ -91,174 +91,158 @@ Website
 
 
 - - -
-
-This help file was dynamically produced by 
-[MarkDoc Literate Programming package](http://www.haghish.com/markdoc/) 
+{it:This help file was dynamically produced by {browse "http://www.haghish.com/markdoc/":MarkDoc Literate Programming package}.}
 ***/
 
+
 * capture program drop generate_race_var
 
 program generate_race_var
```

Comment:
Program seems to give all missing values of `rac1p` to Latino group (in ACS).

Reply:
Thanks, identified issue and will fix. Are there any missing values of `prdtrace` or `pehspnon` in CPS?

Reply:
Just checked again (with `load_data`) and no, only seems to happen with ACS.